doc/source/whatsnew/v1.4.0.rst

```diff
@@ -95,8 +95,10 @@ Other enhancements
 - :meth:`Series.sample`, :meth:`DataFrame.sample`, and :meth:`.GroupBy.sample` now accept a ``np.random.Generator`` as input to ``random_state``. A generator will be more performant, especially with ``replace=False`` (:issue:`38100`)
 - :meth:`Series.ewm`, :meth:`DataFrame.ewm`, now support a ``method`` argument with a ``'table'`` option that performs the windowing operation over an entire :class:`DataFrame`. See :ref:`Window Overview <window.overview>` for performance and functional benefits (:issue:`42273`)
 - :meth:`.GroupBy.cummin` and :meth:`.GroupBy.cummax` now support the argument ``skipna`` (:issue:`34047`)
+- :meth:`DataFrame.between_time` now accepts ``inclusive`` argument as ``{"both", "neither", "left", "right"}`` in place of ``include_start`` and ``include_end`` (:issue:`40245`)
```

Comment:
revert this note (as the deprecation note covers)

Reply:
will do :)